blog.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>adopt-a-dog-spring2019-blog</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+
+  <body>
+    <img class="headercolor" src="images/adoptadog-logo.svg" alt="adoptadog logo-header">
```

Comment:
This isn't in the right spot. The logo should be nested in the header. Also, when creating classes in css you'll want to use kabob casing. Example your html would look similar to 

<img class="header-color" ...>